src/app/projects/states/dashboard/directives/task-dashboard/directives/task-status-card/task-status-card.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="card card-default card-sm card-status">
+  <div class="card-heading task-trigger-dropdown" *ngIf="triggers.length > 0">
+    <mat-form-field style="width: 100%; font-size: 18px;">
+      <mat-select [(value)]="task.status" (change)="triggerTransition(task.status)">
```

Comment:
For `mat-select`, `(change)` isn't calling `triggerTransition`. [The API reference](https://material.angular.io/components/select/api) says it should be `(selectionChange)` instead.

Reply:
Hi,Perry. I just fixed this problem,but below are some unsuccessful things when I push the changes file here.It fails  to run

Reply:
Hey @RayGuo-ergou or @tancnle, why is the branch failing `Node.js CI / build (16.x) (pull_request)` at `npm run lint`? I had similar warnings with mine, but it passed. Thanks!

Reply:
@PerryRose The CI on the pull request trigger seems to checkout [a wrong commit ](https://github.com/thoth-tech/doubtfire-web/runs/6537113961?check_suite_focus=true#step:2:452). It is odd that I can't find that commit hash anywhere 🤔

Reply:
@tancnle Renhao Chen contacted me this issue as well, just warning message but the CI failed, I did not see the CI logs, he just sent me a screenshot.

Reply:
@RayGuo-ergou Renhao's was failing because he had a broken unit test which has been fixed. Could this branch failing because it hasn't been rebased with the eslint changes? @tancnle